src/aat/resources/features/F-053/F-053.feature

```diff
@@ -0,0 +1,103 @@
+    Given a user with [an active profile in CCD]
+    And   a successful call [to create an event token] as in [F-053-Prerequisite]
+    When  a request is prepared with appropriate values
+    And   the request [does not provide valid authentication credentials]
+    And   it is submitted to call the [start case creation as citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-403]
+    And   the response has all other details as expected
+
+  @S-272
+  Scenario: must return negative response HTTP-403 when request provides authentic credentials without authorised access
+    Given a user with [an active profile in CCD]
+    And   a successful call [to create an event token] as in [F-053-Prerequisite]
+    When  a request is prepared with appropriate values
+    And   the request [does not provide an authorised access to the operation]
+    And   it is submitted to call the [start case creation as citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-403]
+    And   the response has all other details as expected
+
+  @S-273
+  Scenario: must return negative response HTTP-422 when request contains an invalid jurisdiction ID
+    Given a user with [an active profile in CCD]
+    And   a successful call [to create an event token] as in [F-053-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid jurisdiction ID]
+    And it is submitted to call the [start case creation as citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [code is HTTP-422]
+    And the response has all the details as expected
+
+  @S-274
+  Scenario: must return negative response HTTP-404 when request contains a non-existing case type ID
+    Given a user with [an active profile in CCD]
+    And   a successful call [to create an event token] as in [F-053-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a non-existing case type ID]
+    And it is submitted to call the [start case creation as citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [code is HTTP-404]
+    And the response has all the details as expected
+
+  @S-275
+  Scenario: must return negative response HTTP-403 when request contains a non-existing user ID
+    Given a user with [an inactive profile in CCD]
+    And   a successful call [to create an event token] as in [F-053-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a non-existing user ID]
+    And it is submitted to call the [start case creation as citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [code is HTTP-403]
+    And the response has all the details as expected
+
```

Comment:
Many scenarios captured originally in the excel sheet are missing. Please first implement them.